### DISCUSSION TITLE
Local Multiplayer in Games

### DISCUSSION THREAD

**Discussion by rebornsunny2000** [skyline-emu/skyline#1313] [General]
I don't know if this is gonna be a stupid question, but here goes:
Can we play local co-op in games that allow it, by connecting 2 controllers to the app, and similarly for games that allow local versus?
Or is it something not possible and I'm just being dumb?

Any answer will be appreciated!!!

**Comment by Lucasbone123** [comment]
Ya es posible, está la opción de agregar mandos en la configuración de Skyline.
